aip/0152.md

```diff
@@ -21,11 +21,11 @@ An API **may** define a `Job` resource to represent a particular task with
 distinct setup, configuration, and execution:
 
 ```proto
-message WritingJob {
+message WriteBookJob {
```

Comment:
Should have some naming convention guidelines here? Perhaps it should be that you should append Job to a valid RPC name? (So <Verb><Noun>Job?)

Reply:
Good idea.